src/libsync/configfile.h

```diff
@@ -72,6 +72,9 @@ class OWNCLOUDSYNC_EXPORT ConfigFile
     /* Force sync interval, in milliseconds */
     quint64 forceSyncInterval(const QString &connection = QString()) const;
 
+    /** Interval within which full local discovery is required, -1 to disable */
+    qint64 fullLocalDiscoveryInterval() const;
```

Comment:
What's the unit?
I know this is not yet your coding style.  But in the future it would be nice to use things like std::chrono::milliseconds as a return type to make it explicit.

Reply:
Agreed!